source/MRViewer/MRUIStyle.h

```diff
@@ -145,6 +146,27 @@ inline bool buttonIconFlatBG( const std::string& name, const Vector2f& iconSize,
     return buttonIconEx( name, iconSize, text, buttonSize, params );
 }
 
+template <typename T>
+bool radioButtonIconFlatBG(
```

Comment:
please add comment

Reply:
please use int instead of <template T>

Reply:
why `flatBG` lets make `radioButtonIcon` and use same colors as now used everywhere